packages/saltcorn-admin-models/models/backup.ts

```diff
@@ -783,6 +784,33 @@ const auto_backup_now_tenant = async (state: any) => {
       await unlink(fileName);
       break;
     //await  SftpClient()
+    case "S3":
+      const s3 = new AWS.S3({
+        accessKeyId: state.getConfig("storage_s3_access_key"),
+        secretAccessKey: state.getConfig("storage_s3_access_secret"),
+        region: state.getConfig("storage_s3_region"),
+      })
+
+      const bucket = state.getConfig("storage_s3_bucket");
+      const s3Key = basename(fileName);
+
+      const fileStream = createReadStream(fileName);
+      const params: AWS.S3.PutObjectRequest = {
+        Bucket: bucket,
+        Key: s3Key,
+        Body: fileStream,
+        ACL: "public-read",
```

Comment:
what is this. Does it need to be a setting - we dont want out backups to be public readable